### PR TITLE
Fix key shortcuts for save/load states.

### PR DESCRIPTION
**Pull request by denisfa**
@rkitover 

We show the wrong description for these shortcuts, although it works
correctly for the keybinding.

Original:
original

New:
new

I will explain why this happens on the code. Fix for #411.

### CODE REVIEW

File: src/wx/guiinit.cpp [hunk 1]
```diff
@@ -2244,11 +2244,12 @@ void MainFrame::add_menu_accels(wxTreeCtrl* tc, wxTreeItemId& parent, wxMenu* me
             // but then if the user overides main menu, that req. is broken..
             wxString txt = (*mi)->GetItemLabelText();
 
-            for (int i = 0; i < 10; i++)
-                if (*mi == loadst_mi[i] || *mi == savest_mi[i]) {
-                    txt = cmdtab[i].name;
```
Comment (denisfa) [inline src/wx/guiinit.cpp]: Here. We are using the wrong index. `cmdtab[i]` is not where the save/load state is. We needed to get the label (`LoadGame01`, for example), then go into the table and check which command has this name and then get the label.

**Comment by denisfa** [comment]
The fact that it is only the number (not the actual description, like `Load saved state 1`) feels very intuitive for me. I know how to do to get this if you prefer, but I would like to see if you like this too.

**Comment by rkitover** [comment]
I think I'd prefer a description like "Load save state in slot 1" and "Save state in slot 1", otherwise no one will know what this means.

**Comment by denisfa** [comment]
@rkitover Here it is. I had a feeling that maybe the previous was enough because the numbers are inside the `Load State` menu. Same for the `Save State`.

new2

Just to confirm this, the text shown is the label from the `*.xrc` files, except the save/load states. This was being done before any of this, and continues to do so. The names are very similar to the labels (the label is usually shorter). Everything is working as expected.

**Comment by rkitover** [comment]
Oh sorry, I missed the part that they were in the `Load state` menu, but this is much better anyway.

**Pull request merged**